src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+
+(click: "really worked at it")[$dissolve[While we’re dancing, I don’t feel anything. I don’t feel the eyes. I don’t hear any hoots, if anyone was even drunk enough to hoot at two eleven year olds. We just twirl and twirl in infinity. I remember thinking: “I hope this lasts forever.”]]
+
+(click: "forever")[$dissolve[And it ends, of course. We’re both breathing heavily and beaming. The guys clap and Leo’s dad waddles over doing a bad copy of our smiles. He says, “Y’all are gonna be heartbreakers one of these days.”]]
+
+(click: "one of these days")[$dissolve[He pinches my cheek, pats my ass. His hands leave long, dark smudges of machine oil. I try to pull away, but he catches me.]]
+
+(click: "catches me")[$dissolve[He says, “Hope you’re nothing like your mother, darling” and loses his balance. He falls ass backwards and his beer’s in the air arcing like a sputtering hose. Everyone busts out laughing, especially Anna.]]
+
+(click: "especially Anna")[$dissolve[Later, I asked Anna how she could laugh. I was so scared, I told her. We shouldn’t hang around those guys anymore. “Nah,” she said, “that’s what it’s gonna be like. You gotta get used to it.”]]
+
+(click: "get used to it")[$dissolve[It wasn’t until years later that I realized that Anna had kicked that motherfucker’s legs out.
+
+(display: "The Gutter - Bathroom - Menu")]]
+
+::The Gutter - Bathroom - Second visit
+I locked myself in the 'Lette's room again with the door stopper. I took a moment to just close my eyes and breathe. Each second of silence was as refreshing and sweet as a big scoop of vanilla ice cream.
+
+(display: "The Gutter - Bathroom - Menu")
+
+::The Gutter - Name prompt ending
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 0)[Right, my name is $playerName. [[Moving on|The Gutter - Ending]].]
+}(else:)[No, [[that's not right|The Gutter - Name prompt ending]].]
+
+(set: $dissolve to (transition: 'dissolve') + (transition-time: 3s))
+(set: $hatchet to true)(set: $multitool to true)
+
+::The Gutter - Ending
+When the night air hit my lungs, the relief was sharper than drinking a cold glass of water after a bong rip. The light of the Gutter’s sign felt like the glow of the north star, revealing my path forward.
+
+>>[[I heard the noise of a familiar assclown behind me and turned to look|The Gutter - Ron's cart finale]].
+>>[[I ripped the envelope from my pocket and devoured Anna’s letter|The Gutter - Anna's letter]].
+
+(set: $gutterEndScenesViewed to 0)(set: $golfCartWreckViewed to false)(set: $annaLetterRead to false)
+
+::The Gutter - Ending hub
+I stood planning my next move in the crisp night air just outside of The Gutter's halo of dirty light.
+
+{(if: $gutterEndScenesViewed is 2)[[[I headed for the Church with all the grim determination I could muster|Church - Intro post-letter]].]}
```

Comment:
Let's also revise these so they don't show empty lines if they're not visible.

Reply:
Revised!